packages/admin-ui/src/switch/switch.style.ts

```diff
@@ -0,0 +1,127 @@
+import { colors, focusVisible } from '@vtex/admin-ui-core'
+import type { StyleProp } from '@vtex/admin-ui-core'
+import { get } from '@vtex/admin-ui-util'
+
+const css = (csx: StyleProp) => csx
+
+const green40 = get(colors, 'green40')
+const green50 = get(colors, 'green50')
+const gray40 = get(colors, 'gray40')
```

Comment:
You can create the following tokens instead:

**Background**
```
bg: {
  form: {
    controlActive: colors.green40
    controlActiveHover: colors.green50,
    controlInactive: colors.gray30,
    controlInactiveHover: colors.gray40,
  }
}
```

Reply:
**Foreground**

```
fg: {
  form: {
    control: colors.white,
  }
}
```

Reply:
Done! Thanks for the direction, @lucasaarcoverde